### PR TITLE
3.1.0

### LINKED ISSUE #122: Speech bubbles are wrong way around
The bubble names don't match the icons

image

### PR DESCRIPTION
**Pull request by johnmgriffith**
• Resolved #122 with speech-bubble and speech-bubble-plus having incorrect names.

• Added feature where you can now search by release number.

**New Icons:**

sigma
percent
x-bar
mu
minimum
maximum
minimum-graph
maximum-graph

image
image

### CODE REVIEW

**Comment by johnmgriffith** [comment]
@paulcpederson , I just saw that if I click the "x" on modal dialogues on this branch, it goes to top again. I made sure that I pulled from master. Know what's going on here?

**Comment by paulcpederson** [comment]
You'll... have to merge the pr to see the changes in `master` 😉

**Comment by johnmgriffith** [comment]
Weird… I just checked master too, and the same "x" sends to top issue is still occurring. Maybe somehow the changes didn't get merged?

File: docs/keywords.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "2d-explore":{
     "alias":[
+      "1.5.0",
```
Comment (paulcpederson) [inline docs/keywords.json:4]: What's the thinking behind adding the version as an alias?

**Comment by paulcpederson** [comment]
> You'll... have to merge the pr to see the changes in master 😉

As I said, you'll need to merge that pr into `master` to see it...

File: docs/keywords.json (hunk 1 above)
Comment (johnmgriffith) [reply docs/keywords.json:4]: We can't search the most recent icons by release number. This helps us know what was added in the current release. We could do something where we added a "new icons" tag to each new release, but this way we don't have to delete those tags in the future.

**Comment by paulcpederson** [comment]
I just merged for y'all, should be in `master` now

**Comment by johnmgriffith** [comment]
> I just merged for y'all, should be in `master` now

Oh I didn't notice that we hadn't merged it yet. Thanks!!

**Comment by KatelynSeitz** [comment]
looks good! 🚀

**Pull request merged**